modules/gsp-cluster/data/values.yaml

```diff
@@ -38,14 +38,6 @@ concourse:
     githubClientId: ${github_client_id}
     githubClientSecret: ${github_client_secret}
     githubCaCert: ${github_ca_cert}
-  worker:
-    replicas: ${concourse_worker_count}
```

Comment:
don't we still want this?

Reply:
I think Alex is right. I don't think concourse will do its own autoscaling therefore it will fallback to the default value of workers which is `2`. We may be OK with that?

Reply:
yeah I wasn't sure how much we cared about this ... worker sizes have grown since the original decision to have 3x workers, I _suspect_ we can get away with two.